CMakeModules/FindPythonLibsNew.cmake

```diff
@@ -0,0 +1,195 @@
+# - Find python libraries
```

Comment:
Is this necessary?

https://cmake.org/cmake/help/latest/module/FindPython.html

Reply:
I think it is. If I try find_package(Python ...) instead, I hit a compilation error when making the pybind module triggered by "#include <Python.h>".

If you can find a way to make it work, though, we can switch. I'd certainly prefer to get rid of the dependency on an external CMake module.